### ISSUE TITLE
Docker Compose version 1.29.2 Incompatible with .env.kibana.token

### ISSUE THREAD

**Issue by ScottFred** [sherifabdlnaby/elastdocker#64]
**Describe the bug**
The README.md says that this repo is compatible with Docker Compose version 1.29.2, but I found that this version of Docker compose is sensitive to white-space in .env files.  This is a known Docker Compose [issue](https://github.com/docker/compose/issues/7624) and has been resolved in later versions.  

When starting elastdocker with `docker-compose up`, a warning message will appear on the command line that says, "WARNING: Python-dotenv could not parse statement starting at line 1".    This happens because the  secrets/.env.kibana.token file has been created during the `make setup` to be written in a way that is not compatible with Docker Compose 1.29.2 because it has whitespace and uses a colon, instead of an equal sign with no white-space.

```
KIBANA_SERVICE_ACCOUNT_TOKEN: <token>
```

If this environment file is re-written in the following way, Docker Compose 1.29.2 is able to read this environment file:
```
KIBANA_SERVICE_ACCOUNT_TOKEN=<token>
```

So, the README.md file should be modified so that it shows that this repo is only compatible with later versions of Docker Compose, or fix the .env.kibana.token file to replace the ": " with an equal sign "=".

**To Reproduce**
1.  Using Docker Compose 1.29.2:  `docker-compose up`
2. See displayed on command line: "WARNING: Python-dotenv could not parse statement starting at line 1".

**Expected behavior**
No WARNING message should be present when starting via `docker-compose up`

**Desktop (please complete the following information):**
 - OS: CentOS 7.x
 - Docker Compose, version 1.29.2

**Comment by sherifabdlnaby** [comment]
Hello, I believe this was fixed by #63 ?

**Comment by ScottFred** [comment]
Sorry!  You're right!  Thanks providing this great repo!
